pkgs/desktops/gnome/extensions/extensionOverridesPatches/freon_at_UshakovVasilii_Github.yahoo.com.patch

```diff
@@ -80,6 +80,6 @@ index 03d469b..6057a3b 100644
  function getSmartData (argv){
 -    const smartctl = GLib.find_program_in_path('smartctl')
 +    const smartctl = GLib.find_program_in_path('@smartmontools@/bin/smartctl')
```

Comment:
While at it, it could be simplified to:

```suggestion
+    const smartctl = '@smartmontools@/bin/smartctl'
```

Reply:
Good point! I've done this change for all replacements.